.tekton/central-db-build.yaml

```diff
@@ -55,18 +55,21 @@ spec:
   - name: clone-fetch-tags
     value: 'true'
 
-  workspaces:
-  - name: git-auth
-    secret:
-      secretName: '{{ git_auth_secret }}'
-
```

Comment:
On one hand I don't mind about these lines being just moved. On the other hand, this may create more work when backporting.